pset-iterators.js

```diff
@@ -42,7 +42,11 @@ Expected Result Shape: A string representing the full name of the president that
 console.log('Problem 1:')
 
 // Add your code below this line
+const firstWhigPresident = presidents.filter(function (president) {
```

Comment:
@deedeemedina nice approach, however, the method to use here was `.find()` since the problem asked for the *first* whig president and `.find()` only returns the first item that meets the test condition inside the callback